jbot-example/pom.xml

```diff
@@ -25,7 +25,30 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
```

Comment:
May I ask why is this dependency needed?

Reply:
It is a dependency coming from spring boot.  This pins it to a non vulnerable but still working version to avoid a CVE for version 1.31 which.